pkg/harvester/mixins/harvester-vm/index.js

```diff
@@ -295,6 +302,9 @@ export default {
 
   async created() {
     await this.$store.dispatch(`${ this.inStore }/findAll`, { type: SECRET });
+    const machineTypes = await this.$store.dispatch('harvester/request', { url: '/v1/harvester/clusters/local?link=machineTypes' });
```

Comment:
Not sure if we can fetch the API using the format `this.$store.dispatch(harvester/findAll`)`.